pkg/agent/openflow/pipeline.go

```diff
@@ -301,24 +330,20 @@ type client struct {
 	roundInfo             types.RoundInfo
 	cookieAllocator       cookie.Allocator
 	bridge                binding.Bridge
-	egressEntryTable      uint8
-	ingressEntryTable     uint8
-	// Flow caches for corresponding deletions.
-	nodeFlowCache, podFlowCache, serviceFlowCache, snatFlowCache, tfFlowCache, mcastFlowCache *flowCategoryCache
-	// "fixed" flows installed by the agent after initialization and which do not change during
-	// the lifetime of the client.
-	gatewayFlows, defaultServiceFlows, defaultTunnelFlows, hostNetworkingFlows []binding.Flow
+
+	featurePodConnectivity *featurePodConnectivity
```

Comment:
Better to not use the same name for an object as the struct?

Reply:
I have named like this: 
```
podConnectivityFeature *featurePodConnectivity
```
This looks strange too.